src/apps/js_v8/js_v8_base.cpp

```diff
@@ -0,0 +1,529 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#include "apps/utils/metrics_tracker.h"
+#include "ccf/app_interface.h"
+#include "ccf/historical_queries_adapter.h"
+#include "ccf/user_frontend.h"
+#include "ccf/version.h"
+#include "crypto/entropy.h"
+#include "crypto/key_wrap.h"
+#include "crypto/rsa_key_pair.h"
+#include "kv/untyped_map.h"
+#include "kv_module_loader.h"
+#include "named_auth_policies.h"
+#include "tmpl/ccf_global.h"
+#include "tmpl/console_global.h"
+#include "tmpl/request.h"
+#include "v8_runner.h"
+#include "v8_util.h"
+
+#include <memory>
+#include <stdexcept>
+#include <vector>
+
+using namespace std;
+using namespace ccf;
+using namespace kv;
+
+namespace ccfapp
+{
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wc99-extensions"
```

Comment:
I believe this was put in to deal with some quirks in quickjs's C code, we hopefully can remove it in this case?

Reply:
Removed.